fake/fake-extension.js

```diff
@@ -885,9 +887,10 @@ var generateMvcRequest = (function() {
 
             // child actions
             if (action.actions) {
+                var that = this;
                 _.forEach(action.actions, function(childAction) {
-                    this.processChildActions(childAction, request, context);
-                }, this);
+                    that.processChildActions(childAction, request, context);
+                });
```

Comment:
Yeah, the Lodash `_.forEach()` changed its API in a backward-incompatible way from 3.x to 4.x, where you can no longer pass in the `this` object with which to bind.  This was breaking the Client fakes.